package.json

```diff
@@ -109,29 +111,27 @@
     "@quarterto/chai-dom-equal": "^1.6.0",
     "@quarterto/chai-selector": "^1.3.0",
     "@quarterto/emphasize-monokai-sheet": "^1.3.0",
-    "@quarterto/gaussian": "^1.0.0",
+    "@quarterto/gaussian": "^1.4.0",
```

Comment:
my gist to npm script doesn't actually do semver 😬 it just increments the minor version on each commit. i don't think this version is backwards compatible. then again, it's only used in [a script to analyse the benchmark results](https://github.com/Financial-Times/google-amp/blob/e62a88e01995e607c5724e4e3461f1447b2bfe3c/scripts/dev-utils/analyse-log.js#L5), which i haven't run in about 18 months, and we should probably do speedcurve to it instead